### PR TITLE
Fix duplicate DAOs when searching using capital characters

### PR DESCRIPTION
**Pull request by tibetsprague**
Fixes https://github.com/daostack/alchemy/issues/1585

### CODE REVIEW

File: src/components/Daos/DaosPage.tsx [hunk 1]
```diff
@@ -67,11 +67,13 @@ class DaosPage extends React.Component<IProps, IState> {
     // If search string greater than 2 search on server for any other DAOs not yet loaded that match this search
     if (searchString.length > 2) {
       const arc = getArc();
+      const firstChar = searchString.charAt(0);
```
Comment (dkent600) [inline src/components/Daos/DaosPage.tsx:70]: Is it not feasible to make the entire DAO name search be case insensitive?

Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:70]: I dont think so

Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:70]: I mean it is if the DAOs are already loaded but for ones we look up on the server I didn't find how to do that with the graphnode

Comment (dkent600) [reply src/components/Daos/DaosPage.tsx:70]: OK I approved, but this looks kinda like a hack.  Maybe could add a code comment explaining the situation?

Comment (tibetsprague) [reply src/components/Daos/DaosPage.tsx:70]: i did below

Comment (orenyodfat) [reply src/components/Daos/DaosPage.tsx:70]: why not lower case the search string and and dao names and then do the search ?

**Pull request merged**